added linux shell script - george.sh

nemothorx opened the pull request:
[attempt 2, now with correct author metadata]

Relatively simple linux shell script added for local use. Timing calculations are different to the JS method, but the end result is within a fraction of a second for the beeps. Future improvements idea:  limit how many loops it runs to suit intermittent running from cron.

Commit:
added linux shell script - george.sh

ryanmonro commented:
Hi @earthnative, I love the idea of a CLI version. I can't get it to run, as it turns out the time command has different parameters in Mac OSX. I don't currently have Linux installed anywhere, so haven't been able to test yet. I'll get back to you when I do.

nemothorx commented:
time command? date? 

but yeah, it's a very first draft script - relies on "mpv" (fork of mplayer) for playing the audio, and I didn't put any fallbacks in. I got it working to local satisfaction and figured other bugs could be resolved as they're encountered! :D 

...It also relies on some possibly non-universal things such as the ability for `sleep` to handle floating point... 

I've got an OSX install I'll poke at it with too :)

ryanmonro commented:
Oh sorry, I meant date.

On Mon, 21 Oct 2019 at 12:31 pm, Nemo Thorx <notifications@github.com>
wrote:

> time command? date?
>
> but yeah, it's a very first draft script - relies on "mpv" (fork of
> mplayer) for playing the audio, and I didn't put any fallbacks in. I got it
> working to local satisfaction and figured other bugs could be resolved as
> they're encountered! :D
>
> ...It also relies on some possibly non-universal things such as the
> ability for sleep to handle floating point...
>
> I've got an OSX install I'll poke at it with too :)
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/ryanmonro/george/pull/2?email_source=notifications&email_token=AFQIXH64C2ASB4JCDFTKCPLQPUA5VA5CNFSM4JBTRIJKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEBYZUXQ#issuecomment-544315998>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFQIXH7G5X32EX27S45X3BLQPUA5VANCNFSM4JBTRIJA>
> .
>

Commit:
updated george.sh in prep for OSX portability

Commit:
half volume and no errors from cron

modified:  george.sh  | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

nemothorx committed:
Merge branch 'ryanmonro:master' into master
